### PR TITLE
Fix build with Bazel >= 2

### PR DESCRIPTION
**Pull request by schoppmp**
This updates dependencies to allow building with Bazel 2 and higher.

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -17,56 +17,41 @@
...
+        "https://github.com/grpc/grpc/archive/v1.27.3.tar.gz",
+    ],
 )
 
-load("@com_github_grpc_grpc//bazel:grpc_deps.bzl", "grpc_deps")
+load("@com_github_grpc_grpc//bazel:grpc_deps.bzl", "grpc_deps", "grpc_test_only_deps")
+
 grpc_deps()
+
+grpc_test_only_deps()
```
Comment (karnseth) [inline WORKSPACE:53]: Could you add a comment what this is needed for?

File: WORKSPACE [hunk 2]
```diff
@@ -17,56 +17,41 @@
...
+
+load("@com_github_grpc_grpc//bazel:grpc_extra_deps.bzl", "grpc_extra_deps")
+
+grpc_extra_deps()
```
Comment (karnseth) [inline WORKSPACE:57]: I assume this is what loads com_google_absl now. Could you add a comment to that effect?

Comment (schoppmp) [reply WORKSPACE:57]: `com_google_absl` is already loaded in `grpc_deps()`, `grpc_extra_deps()` loads transitive dependencies (e.g. Protobuf's dependencies). I added comments to clarify that.

File: WORKSPACE (hunk 1 above)
Comment (schoppmp) [reply WORKSPACE]: I realized it's not needed at all and removed it.

**Comment by karnseth** [comment]
Thanks for the fix!

**Pull request merged**